src/components/pages/EventPage/EventPage.less

```diff
@@ -15,6 +15,7 @@
 .container {
   padding: 1em 0;
   font-size: 16px;
+  min-height: 45vh;
```

Comment:
Because the height of the footer segment is calculated in `em` this will only work for some screens. On my screen, there is still white-space beneath the footer.

![Screenshot](https://image.prntscr.com/image/cGwaeKCwSa_FmlLufQGXfA.png)

![](https://image.prntscr.com/image/cGwaeKCwSa_FmlLufQGXfA.png)